nix/default.nix

```diff
@@ -1,45 +1,44 @@
 { system ? builtins.currentSystem }:
 let
-  pkgs = import ./nixpkgs.nix {};
+  pkgs = (import ./nixpkgs.nix {
+    config = {
+      packageOverrides = pkg: {
+        gpgme = (static pkg.gpgme);
+        libassuan = (static pkg.libassuan);
+        libgpgerror = (static pkg.libgpgerror);
+        libseccomp = (static pkg.libseccomp);
+      };
+    };
+  });
 
-  static = pkg: pkg.overrideAttrs(old: {
-    configureFlags = (old.configureFlags or []) ++
+  static = pkg: pkg.overrideAttrs(x: {
+    configureFlags = (x.configureFlags or []) ++
       [ "--without-shared" "--disable-shared" ];
     dontDisableStatic = true;
     enableSharedExecutables = false;
     enableStatic = true;
   });
 
-  patchLvm2 = pkg: pkg.overrideAttrs(old: {
-    configureFlags = [
-      "--disable-cmdlib" "--disable-readline" "--disable-udev_rules"
-      "--disable-udev_sync" "--enable-pkgconfig" "--enable-static_link"
-    ];
-    preConfigure = old.preConfigure + ''
-      substituteInPlace libdm/Makefile.in --replace \
-        SUBDIRS=dm-tools SUBDIRS=
-      substituteInPlace tools/Makefile.in --replace \
-        "TARGETS += lvm.static" ""
-      substituteInPlace tools/Makefile.in --replace \
-        "INSTALL_LVM_TARGETS += install_tools_static" ""
-    '';
-    postInstall = "";
-  });
-
-  self = {
-    cri-o-static = (pkgs.cri-o.overrideAttrs(old: {
+  self = with pkgs; {
+    cri-o-static = (cri-o.overrideAttrs(x: {
       name = "cri-o-static";
       src = ./..;
-      BUILDTAGS="${old.BUILDTAGS} netgo";
-      EXTRA_LDFLAGS = ''-linkmode external -extldflags "-static -lm"'';
-      dontStrip = true;
-      # DEBUG = 1; # Uncomment this line to enable debug symbols in the binary
```

Comment:
Let's do not remove those, we had a dedicated request for adding debug symbols to the static binary.

Reply:
Seems related to https://github.com/cri-o/cri-o/pull/3529

Shall we simply say `DEBUG = 0;` as default?

Reply:
Yes, sound good to me :)

Reply:
ok will do ;-)